File: zhishu-frontend/src/types/chatSession.ts
import type { QaHistoryScene } from '@/types/qaHistory'

export interface ChatSessionVO {
  conversationId: string
  scene: QaHistoryScene | string
  title?: string
  agentId?: number
  createTime?: string
  updateTime?: string
}

export interface ChatSessionCreateDTO {
  scene?: QaHistoryScene | string
  title?: string
  agentId?: number
}

export interface ChatSessionTitleDTO {
  title: string
}

File: zhishu-backend/zhishu-ai/src/main/java/cn/datafuturex/zhishu/ai/chat/domain/ChatSession.java
package cn.datafuturex.zhishu.ai.chat.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@TableName("chat_session")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChatSession {

    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    @TableField("conversation_id")
    private String conversationId;

    @TableField("user_id")
    private String userId;

    @TableField("scene")
    private String scene;

    @TableField("title")
    private String title;

    @TableField("agent_id")
    private Long agentId;

    @TableField("create_time")
    private LocalDateTime createTime;

    @TableField("update_time")
    private LocalDateTime updateTime;
}
